modules/administration-guide/partials/con_architecture-overview-with-devworkspace.adoc

```diff
@@ -0,0 +1,27 @@
+[id="{prod-id-short}-architecture-overview-with-devworkspace{context}"]
+= {prod-short} architecture with {devworkspace}
+
+include::example$snip_using-devworkspace-warning.adoc[]
+
+.High-level {prod-short} architecture with the {devworkspace} engine
+image::architecture/che-interacting-with-devworkspace.png[]
+
+When {prod-short} is running with the {devworkspace} engine, it runs on three groups of components. The {prod-short} server components and the {devworkspace} engine are the two central server-side components of {prod-short}. Together they manage the User workspaces, which are container-based development environments.
+
+The {prod-short} dashboard controls the desired state of the User workspaces and displays the running workspaces.
+
+The {devworkspace} engine manages the running state of the User workspaces.
+The {devworkspace} engine stops a workspace after 30 minutes of inactivity. This duration is configurable.
+
+The User dashboard and the {devworkspace} engine communicate with valid {orch-name} objects: {devworkspace} Custom Resources.
```

Comment:
Is `{devworkspace} Custom Resources` just the Che name for `{orch-name} objects`? Or are the custom resources a subset of K8s objects? If they are the same thing with a different name in Che, consider revising as follows: `communicate with valid {orch-name} objects, which are referred to as {devworkspace} Custom Resources in Che`.

In any case, revise somehow to replace the colon, which caused confusion for me.

Reply:
https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/